src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/ResourceUtilizationCounters.cs

```diff
@@ -7,24 +7,24 @@ namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring;
 /// Represents the names of instruments published by this package.
 /// </summary>
 /// <remarks>
-/// These counters are currently only published on Linux.
+/// These metrics are currently only published on Linux.
 /// </remarks>
 /// <seealso cref="System.Diagnostics.Metrics.Instrument"/>
 public static class ResourceUtilizationCounters
 {
     /// <summary>
-    /// Gets the CPU consumption of the running application in percentages.
+    /// Gets the CPU consumption of the running application in range <c>[0, 1]</c>.
     /// </summary>
     /// <remarks>
     /// The type of an instrument is <see cref="System.Diagnostics.Metrics.ObservableGauge{T}"/>.
     /// </remarks>
-    public static string CpuConsumptionPercentage => "cpu_consumption_percentage";
+    public static string CpuUtilization => "process.cpu.utilization";
```

Comment:
Be wary that the process semantic conventions aren't stable and might change after you ship this. https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/process-metrics.md#semantic-conventions-for-os-process-metrics. You either need to work with OTel folks to try agreeing on a stable convention up-front, potentially take breaking changes in the future, or have metrics that are unaligned with the standard.

Reply:
@klauco FYI

Reply:
@noahfalk can we have all this functionality marked as experimental + have a clear notice here and in docs that the semantic conventions are not yet stable and we might change this?

Reply:
> @noahfalk can we have all this functionality marked as experimental + have a clear notice here and in docs that the semantic conventions are not yet stable and we might change this?

+1

Reply:
We are free to write whatever text we want in the docs and its np from my perspective if you write that kind of disclaimer. In terms of programatically identifying it as experimental the only thing I am aware of is the ExperimentalAttribute on APIs. There isn't anything that specifically attaches to a metric.

Reply:
I can add this into the instrument's description (e.g. `"This instrument is in Experimental stage and can be changed/removed in future versions."`). What do you think folks?

Reply:
I don't think it would be a substitute for docs, but it wouldn't hurt to do that.

Reply:
Documentation is tracked under #4522, we'll highlight that these metrics are experimental.